tex/tuda-ci.dtx

```diff
@@ -3395,8 +3403,8 @@
 }
 
 \cs_new:cn {g_@@_cc-by-nc-nd-2.0-de:} {
-  Die~Veröffentlichung~steht~unter~folgender~Creative~Commons~Lizenz:\\
-  Namensnennung~--~Keine~kommerzielle~Nutzung~--~Keine~Bearbeitung~ 2.0~Deutschland\\
+  Die~Veröffentlichung~steht~unter~folgender~\glqq{}Creative~Commons\grqq{}-Lizenz:\\
```

Comment:
I'd prefer „“ directly. As the other one requires some package we do not require by default.